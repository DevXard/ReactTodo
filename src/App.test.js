
import { render, fireEvent } from "@testing-library/react";
import App from "./App";

it("renders without crashing", () => {
    render(<App />)
})

it("matches Snapshot", () => {
    const {asFragment} = render(<App />);
    expect(asFragment()).toMatchSnapshot()
})