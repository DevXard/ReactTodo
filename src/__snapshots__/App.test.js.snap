// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <h2>
        Todo List
      </h2>
      <form>
        <label
          for="todo"
        >
          Todo
        </label>
        <input
          id="todo"
          name="todo"
          type="text"
          value=""
        />
        <button>
          Add Todo
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
